CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+**v5.5.1:**
+
+* final_grades.csv: notebook name is now the file name for each row instead of the full path to the notebook
+* final_grades.csv: the file is sorted by notebook name 
+
```

Comment:
Combine these into a single item, suggested rewording:

```
* Updated Otter Grade output CSV to be sorted by notebook name and to use notebook base name instead of full file path as the file name
```